libs/ui/src/invalid_card_screen.test.tsx

```diff
@@ -1,11 +1,71 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 
-import { InvalidCardScreen } from './invalid_card_screen';
+import { InvalidCardScreen, Props } from './invalid_card_screen';
 
-describe('InvalidCardScreen', () => {
-  test('says "Invalid Card"', () => {
-    const { getByText } = render(<InvalidCardScreen />);
-    getByText('Invalid Card');
-  });
-});
+const testCases: Array<{
+  description: string;
+  machine: Props['machine'];
+  reason: Props['reason'];
+  expectedText: string;
+}> = [
+  {
+    description: 'VxAdmin not configured',
+    machine: 'VxAdmin',
+    reason: 'machine_not_configured',
+    expectedText:
+      'This machine is unconfigured and cannot be unlocked with this card. ' +
+      'Please insert a System Administrator card.',
```

Comment:
I decided to start using the new role names (election manager, system administrator) in this component. I'm planning to update existing code in a separate PR ([issue link](https://github.com/votingworks/vxsuite/issues/2181)), so elsewhere in this PR, you'll still see old names (admin, super admin)